src/View.elm

```diff
@@ -20,6 +21,7 @@ view model =
             , Views.Nav.view model
             , main_ [ class "content", role "main" ]
                 [ Views.Content.view model ]
+            , Views.Footer.footerContent
```

Comment:
We only want to show the footer on the home, privacy and supporters pages. Not the others.